src/symtab/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ set(SYMTAB_HEADERS
 # =============================================================================
 add_library(symtab_obj OBJECT ${SYMTAB_SOURCES} ${SYMTAB_HEADERS})
```

Comment:
This confuses me too; why are the headers (including `.hpp`) being added through `add_library`?

Reply:
hmm.. good question.. I am currently not sure. These CMakeFiles need some cleanup...